rascaline-c-api/Cargo.toml

```diff
@@ -15,11 +14,10 @@ bench = false
 [features]
 default = ["chemfiles"]
 chemfiles = ["rascaline/chemfiles"]
-serialization = ["equistore/serialization"]
 
 [dependencies]
 rascaline = {path = "../rascaline", version = "0.1.0", default-features = false}
-equistore = {git = "https://github.com/lab-cosmo/equistore", rev = "467baf0"}
+equistore = {git = "https://github.com/lab-cosmo/equistore", rev = "6c5a833"}
```

Comment:
equistore is fast. It might make it a difference but I think we can try to use the neweset version. Which currently is: `fc4c48d`

Reply:
Should not change anything for rascaline, since all changes between 6c5a and fc4c do not touch the C/Rust API. 

This one is the final set of changes to build system https://github.com/lab-cosmo/equistore/commit/6c5a8333af37b1515fb409394dabe6f84ca0a858